<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 154, 38);
background-image: url(:/resource/photo/Resource/2.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;新宋体&quot;;
</string>
   </property>
   <property name="text">
    <string>工资管理系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0);
background:transparent;
font: 12pt &quot;仿宋&quot;;</string>
   </property>
   <property name="text">
    <string>不一致</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>账    号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_accout">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>密    码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Passwword">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>确认密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ConfirmPassword">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>名    字：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>部    门：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_departmen">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>岗    位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_position">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>地    址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_address">
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QRadioButton" name="radioButton_Worker">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>员工注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_manager">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>管理员注册</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="pushButton_register">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_position</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_departmen</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_ConfirmPassword</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Passwword</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_accout</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_position</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_departmen</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_ConfirmPassword</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Passwword</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_accout</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
